activerecord/lib/active_record/migration/compatibility.rb

```diff
@@ -102,19 +102,6 @@ class << t
           end
         end
 
-        def change_table(table_name, options = {})
-          if block_given?
-            super(table_name, options) do |t|
-              class << t
```

Comment:
Thanks for your kind explanation,  I understood that well. It's just as you say.
I'll close this PR. Thank you again.